packages/react-components/src/tooltip/src/Tooltip.tsx

```diff
@@ -1,27 +1,30 @@
 import "./Tooltip.css";
 
+import { ComponentProps, ElementType, ForwardedRef, ReactNode } from "react";
 import { Text } from "../../text";
-import { any, elementType, func, oneOfType, string } from "prop-types";
-import { forwardRef } from "react";
-import { mergeProps } from "../../shared";
+import { forwardRef, mergeProps } from "../../shared";
 
-const propTypes = {
+interface InnerTooltipProps {
     /**
      * An HTML element type or a custom React element type to render as.
      */
-    as: oneOfType([string, elementType]),
+    as?: ElementType;
     /**
      * React children.
      */
-    children: oneOfType([any, func]).isRequired
-};
+    children: ReactNode | Function
```

Comment:
This is passed to Text, wich pass it to Box, but nobody uses a Func. is there something i'm not seeing?